### DISCUSSION TITLE
Which VScode extension to load for Arduino dev?

### DISCUSSION THREAD

**Discussion by Fran-3** [microsoft/vscode-discussions#727] [Extension Development QnA] [answered]
[One Video Tutorial](https://www.youtube.com/watch?v=WVZxK2MEbE4) says to just load the Platform IO IDE Extension and away you go.

[Another Video Tutorial](https://www.youtube.com/watch?v=WVZxK2MEbE4) says to download & install VScode,
then download & install the Arduino IDE...
then install the Arduino Extension in VScode
then install the C++ extension in VScode

So I'm confused.

I actually already had the Arduino IDE on my PC when I downloaded and installed VScode.
So when I look for extensions I see both...
the Platform IO IDE Extension
and the Arduino IDE Extension

Should I install both or which one?
And do I need to install the C++ extension?
In the first Video Tutorial he just installs Platform IO IDE and then starting writing code.

Thanks for any help.

PS - If I am in the wrong Forum please advise where to post this. Thanks.

**Comment by gjsjohnmurray** [comment, marked as answer]
This forum is for people who are developing VS Code extensions. I think your question belongs in an Arduino forum. Or perhaps direct it to the authors of the extension(s) you refer to.

**Reply by Fran-3** [reply]
Thanks. Is there no forum for users of VScode? I'm sure I'll have more questions, not all relative to Arduino. Thanks.
